src/consensus/pbft/pbft.h

```diff
@@ -370,11 +487,110 @@ namespace pbft
       switch (serialized::peek<PbftMsgType>(data, size))
       {
         case pbft_message:
+        {
           serialized::skip(data, size, sizeof(PbftHeader));
           message_receiver_base->receive_message(data, size);
           break;
-        default:
-        {}
+        }
+        case pbft_append_entries:
+        {
+          AppendEntries r;
```

Comment:
This could move to its own function.

Reply:
https://github.com/microsoft/CCF/issues/753